var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -262,7 +262,7 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
         patch('darwin/gcc-4.9.patch2', when='@4.9.0:4.9.3')
 
     patch('piclibs.patch', when='+piclibs')
-    patch('gcc-backport.patch', when='@4.7:4.9.2,5:5.3')
+    patch('gcc-backport.patch', when='@4.7:4.9.3,5:5.3')
```

Comment:
might also apply for 4.9.4 - I'll check

Reply:
Apparently it does not apply to 4.9.4. Is this good to merge from your side?